src/app/views/users/_user.html.erb

```diff
@@ -11,8 +11,12 @@
           <%= link_to user.name, user_functions_path(user) %>
         </p>
         <p class="subtitle is-6">
-          @<%= user.username %>
+          <%= link_to user_github_url(user),  target: "_blank" do %>
```

Comment:
we'll need `rel='nooperner'` here 
similar to the markdown parser here: 
https://github.com/emad-elsaid/rubyfunctions/blob/master/src/app/extensions/html_from_markdown.rb#L34

```suggestion
          <%= link_to user_github_url(user),  target: "_blank" do %>
```

Reply:
can't relate !!

Reply:
You may need to read this article here to get the full context 
https://developers.google.com/web/tools/lighthouse/audits/noopener
and as this url will open github (external website) then we'll need to ad `rel` attribute to it as the article mentions